Resources/Locales/fr_fr/openspades.po

```diff
@@ -445,7 +446,9 @@ msgstr "Nom de joueur"
 #: Resources/Scripts/Gui/CreateProfileScreen.as:104
 msgctxt "CreateProfileScreen"
 msgid "You can change it later in the Setup dialog."
-msgstr "Vous pourrez le changez par la suite via la boîte de dialogue de Configuration"
+msgstr ""
```

Comment:
Not sure about this one, but might be an extra newline before the start of the translations